pallets/loans/src/lib.rs

```diff
@@ -804,17 +825,13 @@ impl<T: Config> Pallet<T> {
     ) -> DispatchResult {
         let exchange_rate = Self::exchange_rate(currency_id);
         let collateral = Self::calc_collateral_amount(redeem_amount, exchange_rate)?;
-        AccountCollateral::<T>::try_mutate(
-            currency_id,
-            who,
-            |collateral_balance| -> DispatchResult {
-                let new_balance = collateral_balance
-                    .checked_sub(collateral)
-                    .ok_or(Error::<T>::Underflow)?;
-                *collateral_balance = new_balance;
-                Ok(())
-            },
-        )?;
+        AccountDeposits::<T>::try_mutate(currency_id, who, |deposits| -> DispatchResult {
```

Comment:
shall we put `is_collateral` to false, if `vocher_balance` drops down to zero? because now in frontend if users redeem all then they have to click the button again to disable collateral